File: source/server/server/song.cpp
#include "song.h"

Song::Song(int id, std::string name, char* image, int artist_id, int album_id)
{
	this->id = id;
	this->name = name;
	this->image = image;
	this->artist_id = artist_id;
	this->album_id = album_id;
}

int Song::get_id()
{
	return id;
}

std::string Song::get_name()
{
	return name;
}

char* Song::get_image()
{
	return image;
}

int Song::get_artist_id()
{
	return artist_id;
}

int Song::get_album_id()
{
	return album_id;
}

File: source/desktop_client/backend/src/song.py
class Song:
    def __init__(self, id, name, genre, artist_id, album_id, length, streams):
        self.id = id
        self.name = name
        self.genre = genre
        self.artist_id = artist_id
        self.album_id = album_id
        self.length = length
        self.streams = streams

    def get_name(self):
        return self.name

    def get_id(self):
        return self.id

    def get_genre(self):
        return self.genre

    def get_artist_id(self):
        return self.artist_id

    def get_album_id(self):
        return self.album_id

    def get_length(self):
        return self.length

    def get_streams(self):
        return str(self.streams)